controllers/wizard/fileUpload/form/SubmissionFilesUploadForm.inc.php

```diff
@@ -177,10 +179,23 @@ function execute() {
 				'username' => $user->getUsername()
 			)
 		);
+		
+		$this->_submissionFile = $submissionFile;
```

Comment:
I think these changes might be a little less convoluted if you passed `$submissionFile` as a parameter to `parent::execute`; where the hook is called, `func_get_args` is used to pass any incoming parameters into the hook.

Reply:
Deconvoluted.